README.md

```diff
@@ -91,7 +81,7 @@ To make fields private, just give them a name starting with `#`. A shorthand for
 
 By defining things which are not visible outside of the class, ESnext provides stronger encapsulation, ensuring that your classes' users don't accidentally trip themselves up by depending on internals, which may change version to version.
 
-Note that ESnext provides private fields only as declared up-front in a field declaration; private fields cannot be created as expandos.
+Note that ESnext provides private fields only as declared up-front in a field declaration; private fields cannot be created as expandos.??
```

Comment:
Oh sorry, that term must be specific for implementers! It's supposed to mean, other properties that are added outside of the constructor, literal or initial code that sets up the instance (yes, it's vague and implementation-dependent). How about this for a clearer wording?

> private fields cannot be created as additional properties added later.